src/com/facebook/buck/android/AndroidResource.java

```diff
@@ -387,6 +396,29 @@ public Sha1HashCode getAbiKeyForDeps() {
         .filter(NON_EMPTY_RESOURCE);
   }
 
+  private ImmutableList<Path> getResourceDirs() {
+    switch (resourceDependencyMode) {
+      case FIRST_ORDER:
+        return ImmutableList.of(getResolver().getPath(Preconditions.checkNotNull(getRes())));
+      case TRANSITIVE:
+        return FluentIterable.from(getNonEmptyResourceDeps())
+            .append(this)
+            .transform(
+                new Function<HasAndroidResourceDeps, Path>() {
+                  @Override
+                  public Path apply(HasAndroidResourceDeps input) {
+                    return Optional.fromNullable(input.getRes())
+                        .transform(getResolver().getPathFunction())
+                        .get();
```

Comment:
You have an `get()` call without checking presence.  I did this instead:

```diff
-  .get();
+  .orNull();
```

And then...